lib/tarantool_server.py

```diff
@@ -735,7 +735,7 @@ def kill_current_test(self):
     def wait_stop(self):
         self.process.wait()
 
-    def stop(self, silent=True):
+    def stop(self, silent=True, signal=None):
```

Comment:
It would be better to receive signal.SIG* constants here instead of strings. We can also set signal.SIGTERM as default. process.send_signal(signal.SIGTERM) is the alias for process.terminate() (see [here]), so this will not change the behaviour, but will shrink the code.

[1]: https://docs.python.org/2/library/subprocess.html#subprocess.Popen.send_signal

Reply:
I should have to clarify that it is good to provide ability to give a signal as a string for the test-run's command to provide better support for different platforms. However stop() is the internal function and so should be called from test-run itself with signal.SIG* as an argument value.

Changed the commit accordingly.